### DISCUSSION TITLE
A readme project for learning Git and GitHub

### DISCUSSION THREAD

**Discussion by ibrahimraimi** [devstrons/support#32] [Ideas]
This readme project will help beginners to learn and practice Git and GitHub. It will include introductory content, practicals or exercises, guides, tutorials, and how-tos, all easy to understand.

**Comment by aminoxix** [comment]
Thanks @ibrahimraim, to raise this point.

We'll look into it.

But, if you wanna work on any of the particular things you mentioned above, I'll give you the authority to take it from your end.

**Reply by ibrahimraimi** [reply]
Yes, @aminoxix I would love to work on this particular project.

**Reply by aminoxix** [reply]
Sure, go ahead! 🚀 

Have you joined our [discord server](https://dsc.gg/devstrons)?

**Reply by ibrahimraimi** [reply]
Yes, have joined the discord server :)

**Reply by aminoxix** [reply]
Cool, you can raise pull requests, will happy to review :) 

Thanks again!

**Reply by aminoxix** [reply]
@ibrahimraimi: Kindly let us know your discord username [here](https://github.com/devstrons/support/issues/24)

**Comment by scriptyvijay** [comment]
A good _initiative_ would love to contribute and learn

**Reply by aminoxix** [reply]
Thanks @scriptyvijay to show your interest, feel free to join our discord server for any queries.

**Comment by tusqasi** [comment]
Seems cool. How do we go about doing this?  
What would be the goals and outcomes for this?

**Reply by aminoxix** [reply]
@ibrahimraimi may justify this.

**Reply by tusqasi** [reply]
I will start with a example.

1. What is git? A small explanation. What is github/gitlab?
3. List of resource to start with git. Links to videos, articles, guides etc.
4. Example workflows. Showing how someone uses git.
5. Maybe a hands on workshop sometime.
6. Working with github/gitlab.
7. Working with others on a project.

After going through this guide/course. The learner would be able to :
1. Use git for their own projects.
2. Use github, to contribute to other projects
3. Help others.

What do you guys say about this? Is this looking good? What can change here?

**Reply by aminoxix** [reply]
Awesome @tusqasi, thanks for the info. 💙 

@ibrahimraimi let's look up the way what's suggested above.

I think we would create this in a way, so can create a quiz out of it; that may be used in our official app of @devstrons.
